Build: highlight the name of the version

humitos opened the pull request:
I find pretty hard to know what's the version of the current build on the build details' page, so I want to make it more visible.

I'm following the same style/pattern we are following for the build id and I like it.

![Screenshot_2025-07-07_12-50-18](https://github.com/user-attachments/assets/facf70e5-7d38-485f-b74f-c94b8b3e8186)

agjohnson commented on readthedocsext/theme/templates/builds/includes/build_name.html:
```diff
@@ -50,7 +50,9 @@
-        Version {{ version_name }}
+        Versions
+        <span class="divider">/</span>
+        <code class="ui grey text">{{ version_name }}</code>
```
I think this works broken out into a separate breadcrumb, but:

- Don't use `.ui.text` to change link colors, it only affects the inactive link color. If you hover that link it will turn the hover blue that links normally do. I would use the default styling instead of trying to force the section to stand out.
- You should only translate the bare minimum, don't write HTML blocks in `blocktrans`
- `divider` elements should be sibling to `section` elements, not nested. I'm guessing there might be some display issues with this.
- I wouldn't use `code` here, we don't do that for anything else in the breadcrumb

humitos replied:
Cool. I will follow these rules and update the PR. I should probably have marked this PR as Draft and request for early feedback. I wanted to know if we were happy with idea more than the technical side.

humitos committed:
Build: highlight the name of the version

I find pretty hard to know what's the version of the current build on the build
details' page, so I want to make it more visible.

I'm following the same style/pattern we are following for the build id and I
like it.

humitos committed:
Lint and format

agjohnson commented:
I didn't think about the listing view, I don't think this quite works there. As a breadcrumb, `Version` has gone from a descriptor of the version type to just a navigation item.

The reason the verbose name of the version type is in there is to differentiate between versions and pull request builds:

![image](https://github.com/user-attachments/assets/2c7ddf67-4f61-4d99-8879-e8a687bc73df)

What you have here is going to make one style for versions (`Versions / {name} / #1234`) and a separate style for pull requests (`Pull request #999 / #1235`). I think we should stick with consistency here and not use the extra breadcrumb section in the listing view -- that is, keep what we have now on the listing view.

The header change mostly works on the build detail view though, but I'd suggest we always have the extra `Versions` breadcrumb for navigation:

- `Versions > {name} > Builds > #1234`
- `Versions > Pull request 999 > Builds > #1235`

humitos committed:
Feedback

humitos committed:
Link breadcrums properly

humitos committed:
Updates for `page_title`

humitos commented on readthedocsext/theme/templates/builds/includes/build_name.html:
```diff
@@ -33,51 +33,51 @@
+        <a href="{% url "projects_detail" build.project.slug %}?slug={{ build.get_version_slug }}">
+        {% endif %}
+
+        {{ build.get_version_name }}
```
Updated 👍🏼

agjohnson commented:
![image](https://github.com/user-attachments/assets/d6ee29df-4943-4bf6-81ff-e1fb01cbf88d)

The build listing is still off. I don't think we should put a breadcrumb like this in the build list, from my comment here:

- https://github.com/readthedocs/ext-theme/pull/625#issuecomment-3049917495

For consistency that should still be `Version latest` and `Pull request 999`. It's too noisy with the `Versions` breadcrumb in every line.

agjohnson reviewed (commented):
And some small adjustments, my main feedback is still at:

- https://github.com/readthedocs/ext-theme/pull/625#issuecomment-3053937982

humitos commented:
> For consistency that should still be `Version latest` and `Pull request 999`. It's too noisy with the `Versions` breadcrumb in every line.

I tend to agree here. However, the motivation of this PR was to visually distinct the word "Version" from the name of the version itself. So, if we are removing the breadcrumb here I would like to use a different color at least for the name of the version.

An idea:

![Screenshot_2025-07-10_11-52-51](https://github.com/user-attachments/assets/7299dd40-88d7-4bbc-9215-14d95128cbf3)

* It uses `code` for the version name
* It uses grey color for it

I want to highlight the name of the version somehow over the other elements of that row.
